bin/verilator

```diff
@@ -582,9 +595,16 @@ clock signals directly.  This should be transparent to the user.
 
 Generates a script for the specified make system.
 
-Supported make systems are gmake and cmake. Both can be specified.
+Supported make systems are gmake, make, and cmake. This option specifies
+which command is used to build. Both gmake and make means GNU make.
+If gmake is specified, system("gmake -f ... -C ...") is called to build.
+If make is specified, system("make -f ... -C ...") is executed instead.
```

Comment:
For gmake/make we should read MAKE from the environment (add a getenvMAKE()) and call that.  If not in the environment then it defaults to gmake or make.

Then also need documentation on MAKE in bin/verilator.  I don't think we need to put MAKE into the makefiles as it will be in the environment already.

Reply:
Now environment variable MAKE is used.
https://github.com/verilator/verilator/blob/0f96a4fa7dc72668d641ef18d31665edb9c896f7/src/V3Options.cpp#L437-L439

I agree that MAKE should not be in the makefile because the variable is special in GNU make for their job server feature.